### DISCUSSION TITLE
Welcome to hearingaid-prototype Discussions!

### DISCUSSION THREAD

**Discussion by m-r-s** [m-r-s/hearingaid-prototype#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gailbricks1** [comment]
I am new to the GitHub community. I am interested in improving hearing aid based on my husband’s experience. He has phonax. He does not wear them often. According to him, they don’t fit well in his ears. Additionally, the hearing aids pick-up background noise.

**Reply by m-r-s** [reply]
It is great that you are willing to help.
Currently, this project is paused, because there is somehow a conflict of interests with my employer, where I work on the same goals.

Your husbands problems with his hearing devices are common.
If the problems cannot be solved by his current hearing care professional, he might try to get a second opinion elsewhere.
